src/server/set_family_test.cc

```diff
@@ -16,7 +16,7 @@ extern "C" {
 #include "redis/zmalloc.h"
 }
 
-ABSL_DECLARE_FLAG(bool, legacy_saddex_keepttl);
+ABSL_DECLARE_FLAG(std::string, shard_round_robin_prefix);
```

Comment:
@abhijat do you remember why we need legacy_saddex_keepttl ?

Reply:
it was not set or used :thinking: